databuilder/extractor/restapi/rest_api_extractor.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import importlib
+from typing import Iterator, Any  # noqa: F401
+
+from pyhocon import ConfigTree  # noqa: F401
+
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.rest_api.base_rest_api_query import BaseRestApiQuery  # noqa: F401
+
+
+REST_API_QUERY = 'restapi_query'
+MODEL_CLASS = 'model_class'
+
+# Static record that will be added into extracted record
+STATIC_RECORD_DICT = 'static_record_dict'
```

Comment:
what is an example value?

Reply:
Current use case for Mode is like below:
`'extractor.mode_dashboard.extractor.restapi.{}'.format(STATIC_RECORD_DICT): {'product': 'mode'},`

Let me actually add this into ModeDashboardExtractor.